modules/standard/Path.chpl

```diff
@@ -610,6 +624,95 @@ proc file.realPath(out error: syserr): string {
   return "";
 }
 
+// Compute the common prefix length between two lists of path components.
+private proc commonPrefixLength(a1: [] string, a2: [] string): int {
+  // Can we use the builtin min/max functions for this?
+  var (min, max) = if a1.size < a2.size then (a1, a2) else (a2, a1);
```

Comment:
Let's make this a `const ref (min, max)` to create a read-only reference to the arrays, rather than copying them.

Also, I'd rename `min`/`max` variables, since they collide with `min()` and `max()` built-ins.

I'm terrible with names, but here's some ideas:

- `small`, `big`
- `minPath`, `maxPath`
- `A`, `B`